app/build.gradle

```diff
@@ -0,0 +1,32 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 25
+    buildToolsVersion "26.0.2"
+    defaultConfig {
+        applicationId "com.wherecycle.smartrecycle"
+        minSdkVersion 21

```

Comment:
Ok, so we're initially supporting Android 5.0+ only?

Reply:
Yeah, as you suggested I'd rather avoid multi dex issues.